functions/controllers/user/imageUpload.js

```diff
@@ -0,0 +1,56 @@
+const Busboy = require('busboy')
+const path = require('path')
+const os = require('os');
+const fs = require('fs')
+const config = require('../../util/config')
+const { admin } = require('../../util/admin')
+const { status, message } = require("../../util/constants");
+const HttpStatus = require("http-status-codes");
+
+config.storageBucket
+
+const imageUpload = async (req, res, db) => {
+    const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = HttpStatus
+    const { error, success } = status
+    const { wrongFileSubmitted, somethingWentWrong, imageUpdateSucces } = message
+
+    const busboy = new Busboy({headers: req.headers});
+    let imageFileName
+    let imageToBeUploaded = {};
```

Comment:
Why are you using let instead of const

Reply:
it has to be let because i use them outside of that scope.. i tried using const but the eslint rules. ban it and i was having errors